### PR TITLE
Add opencollective button

### PR DESCRIPTION
**Pull request by Jc-L**
Attempt for #117. FontAwesome has currently no Open Collective icon (however there is an [issue](https://github.com/FortAwesome/Font-Awesome/issues/14475) for this). So I hard-coded an SVG icon with the needed styles. Current version does not seem really maintainable to me.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -70,6 +70,23 @@ <h2>Contact Us</h2>
                 Reach out to us on Twitter, GitHub, or join us in the chat room.
             </p>
             <ul class="list-inline banner-social-buttons">
+                <li>
+                    <a href="//twitter.com/{{ site.twitter_username }}" class="btn btn-default btn-lg">
```
Comment (jancborchardt) [inline index.html]: This should be `opencollective.com`, right? :)

File: index.html [hunk 2]
```diff
@@ -70,6 +70,23 @@ <h2>Contact Us</h2>
...
+                                          <path d="M34.6194245,8.17880011 C36.7508489,11.2495266 38,14.9788859 38,19 C38,23.0211141 36.7508489,26.7504734 34.6194245,29.8211999 L29.6976779,24.8994533 C30.6641742,23.1506053 31.2142857,21.1395454 31.2142857,19 C31.2142857,16.8604546 30.6641742,14.8493947 29.6976779,13.1005467 L34.6194245,8.17880011 Z" id="closing-o" fill="#B8D3F4"></path>
+                                      </g>
+                                  </g>
+                              </g>
+                          </svg>
+                        </i> <span class="network-name">OpenCollective</span>
```
Comment (jancborchardt) [inline index.html]: Open Collective is written with a space inbetween. :)

**Comment by Jc-L** [comment]
Should be good for the two comments, but the final layout ends up creating two rows of buttons without space. Currently there is no specific css for this, only the bootstrap `.list-inline>li` style. Do you have some convention to organize scss/css content if I want to change this as well ?

**Comment by jancborchardt** [comment]
> Should be good for the two comments, but the final layout ends up creating two rows of buttons without space. 

I’d say let’s simply remove the "Join us" button then, as commented in https://github.com/opensourcedesign/opensourcedesign.github.io/issues/110#issuecomment-474430616 – what do you think?

**Comment by jancborchardt** [comment]
@Jc-L could I also ask you to squash your commits into one? That way we have a cleaner history in case we need to look for changes or errors later. :)

**Comment by evalica** [comment]
@jancborchardt the person that applies the PR can use the squash feature form GitHub. No need from the contributor to do it.

**Comment by jancborchardt** [comment]
> @jancborchardt the person that applies the PR can use the squash feature form GitHub. No need from the contributor to do it.

Ooooh love it, never used that before. Thank you! :)

**Comment by RDIL** [comment]
I usually squash all my PRs.

**Pull request merged**